src/signal.rs

```diff
@@ -2093,3 +2094,216 @@ mod cell_area {
+                connect(self.to_glib_none().0, "accel-cleared",
+                    transmute(trampoline as usize), Box::into_raw(f) as *mut _)
+            }
+        }
+
+        fn connect_accel_edited<F>(&self, accel_edited_func: F) -> u64
+        where F: Fn(&Self, TreePath, u32, ModifierType, u32) + 'static {
+            unsafe {
+                let f: Box<Box<Fn(&Self, TreePath, u32, ModifierType, u32) + 'static>> =
+                    Box::new(Box::new(accel_edited_func));
+                connect(self.to_glib_none().0, "accel-edited",
+                    transmute(e_trampoline as usize), Box::into_raw(f) as *mut _)
+            }
+        }
+    }
+
+    unsafe extern "C" fn trampoline(this: *mut GtkCellRendererAccel,
+                                    path: *const c_char,
+                                    f: &Box<Fn(&CellRendererAccel, TreePath) + 'static>) {
+        callback_guard!();
+        let path = from_glib_full(gtk_tree_path_new_from_string(path));
+        f(&CellRendererAccel::from_glib_none(this), path);
+    }
+
+    unsafe extern "C" fn e_trampoline(this: *mut GtkCellRendererAccel,
+                                      path: *const c_char,
+                                      accel_key: c_uint,
+                                      accel_mods: GdkModifierType,
+                                      hardware_keycode: c_uint,
+                                      f: &Box<Fn(&CellRendererAccel, TreePath, u32, ModifierType, u32) + 'static>) {
+        callback_guard!();
+        let path = from_glib_full(gtk_tree_path_new_from_string(path));
+        f(&CellRendererAccel::from_glib_none(this),
+           path,
+           accel_key,
+           accel_mods,
+           hardware_keycode);
+    }
+}
+
+pub trait CellRendererComboSignals {
+    fn connect_changed<F>(&self, changed_func: F) -> u64
+        where F: Fn(&Self, TreePath) + 'static;
+}
+
+mod cell_renderer_combo {
+    use CellRendererCombo;
+    use TreePath;
+    use std::mem::transmute;
+    use ffi::GtkCellRendererCombo;
+    use glib::object::Downcast;
+    use glib::signal::connect;
+    use glib::translate::*;
+    use ffi::gtk_tree_path_new_from_string;
+    use libc::c_char;
+
+    impl super::CellRendererComboSignals for CellRendererCombo {
+        fn connect_changed<F>(&self, accel_cleared_func: F) -> u64
+        where F: Fn(&Self, TreePath) + 'static {
+            unsafe {
+                let f: Box<Box<Fn(&Self, TreePath) + 'static>> =
+                    Box::new(Box::new(accel_cleared_func));
+                connect(self.to_glib_none().0, "changed",
+                    transmute(trampoline as usize), Box::into_raw(f) as *mut _)
+            }
+        }
+    }
+
+    unsafe extern "C" fn trampoline(this: *mut GtkCellRendererCombo,
+                                    path: *const c_char,
+                                    f: &Box<Fn(&CellRendererCombo, TreePath) + 'static>) {
```

Comment:
The `GtkTreeIter*` argument seems to be missing: https://developer.gnome.org/gtk3/stable/GtkCellRendererCombo.html#GtkCellRendererCombo-changed